### PR TITLE
docs: Fixing missing 'c' in installation guide documentation

### PR DESCRIPTION
**Pull request by JordanTB**
Fixed missing 'c' in 'docker-compose'

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by sfirke — COMMENTED** [review]
Thanks! This should actually be docker compose, would you mind removing the hyphen? This was introduced accidentally in https://github.com/apache/superset/pull/30030

**Comment by sfirke** [comment]
Oh and the failing check says you should edit your PR title:
>  Error: Please format your PR title to match: `^(build|chore|ci|docs|feat|fix|perf|refactor|style|test|other)(\(.+\))?(\!)?:\s.+`!

File: docs/docs/installation/docker-compose.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 :::caution
 Since `docker compose` is primarily designed to run a set of containers on **a single host**
 and can't support requirements for **high availability**, we do not support nor recommend
-using our `docker- ompose` constructs to support production-type use-cases. For single host
+using our `docker-compose` constructs to support production-type use-cases. For single host
```
Comment (michael-s-molina) [inline docs/docs/installation/docker-compose.mdx:18]: `docker-compose` is being deprecated 😄 
```suggestion
using our `docker compose` constructs to support production-type use-cases. For single host
```

Comment (rusackas) [reply docs/docs/installation/docker-compose.mdx:18]: @mistercrunch [I already did that](https://github.com/apache/superset/pull/30030)... and it was my fat fingers that screwed up this particular instance ;)

**Review by rusackas — APPROVED** [review]

**Pull request merged**